site/gatsby-site/playwright.config.ts

```diff
@@ -75,7 +75,7 @@ export default defineConfig({
 
   /* Run your local dev server before starting the tests */
   webServer: {
-    command: 'NODE_OPTIONS=--max-old-space-size=5600 npm run serve',
+    command: 'npx -y pm2 start npm --name "web-server" -- run serve && npx pm2 logs "web-server"',
```

Comment:
This line is where pm2 is used, should help with restarting the local server when it crashes